guides/17_Migration_to_3_0.md

```diff
@@ -60,22 +71,98 @@ change it to
 await messagingSession.start();
 ```
 
+### `MeetingSessionPOSTLogger` to `POSTLogger`
+
+With Amazon Chime SDK for JavaScript v3, we have removed `MeetingSessionConfiguration` dependency from the `MeetingSessionPOSTLogger`. The `MeetingSessionPOSTLogger` is now renamed to `POSTLogger`.
+
+Builders can add optional fields to `POSTLogger` which will be included when `POSTLogger` sends the HTTP POST logs to builder provided URL:
+
+1. HTTP POST request `headers`.
+2. Informational `metadata`. For example, `meetingId`, `attendeeId`, `appName`. This `metadata` depends on builders and `POSTLogger` will include this `metadata` as part of the request body when making the HTTP POST requests to builder provided URL.
+
+#### Constructor
+
+Before in 2.x:
```

Comment:
Kyu helped me update mine in the device controller PR but he grouped the before and after in one code block and only use // Before and // After so you can share the same common code such as meetingSessionConfiguration

Reply:
Yup makes sense, so updated my docs with this pattern.